src/__tests__/core/block.transactionFilter.test.ts

```diff
@@ -89,6 +89,78 @@ describe('transactionFilter on Block->transactions', async () => {
     expect(result.errors).toHaveLength(1);
     expect(result.errors[0].message).toMatch(/^Expected type TransactionFilter/);
   });
+
+  test('contractCreation: true value, returns only transactions with contract creations', async () => {
```

Comment:
Could you add some tests to cover the different permutations of the filtering arguments? `withInput` and `contractCreation`.

Reply:
Sure - adding